zapier/zapier-platform-core
# Added call to console.log or console.error

## Description
Adding a call to `console.log` (or `console.error`) in addition to what `z.console.log` already does.

## Type of Changes

|   | Type |
| ------------- | ------------- |
|   | :bug: Bug fix  |
| ✓  | :sparkles: New feature |
|   | :hammer: Refactoring  |
|   | :link: Update dependencies |
|   | :scroll: Docs |


## Related Issue

[Takes care of PDE-70](https://zapierorg.atlassian.net/browse/PDE-70)

## Before / Current Behavior
Currently does not log to console.log

## After / New Behavior
Logs to console.log

## Testing Instructions
Didn't see any unit tests - happy to add some - but here's how I tested things in the node console
```js
const injectZObject = require('./src/app-middlewares/before/z-object');
const z = injectZObject({ _zapier: { promises: [], logger: () => {} } }).z
z.console.log('I will be console logged')
```

## TODOs
